.circleci/config.yml

```diff
@@ -49,6 +49,26 @@ commands:
       - run: git branch -f stable HEAD
       - run: git push origin stable
 
+  build_and_deploy_docs_to_github_pages:
+    description: Build Sphinx documentation and push GitHub stable branch which points to the latest relased tag/commit
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "b6:94:61:c3:e1:8f:25:e5:43:6d:51:9e:43:c4:16:11"
+      - common_setup
+      - checkout
+      - run: make docs
+      - run: git worktree add gh-pages gh-pages
+      - run: cd gh-pages
```

Comment:
This is a no-op since CircleCI executor is stateless. Any commands after this line are executed at the root of the checkout, making a mess.

Reply:
Aha, yes, I see that every new `run` starts a new process. But one possible solution we can use is the multi-line syntax for the run command which will prevent these commands from being forgotten.